jenkins-dcp.planx-pla.net/manifest.json

```diff
@@ -62,7 +62,7 @@
     "environment": "qaplanetv1",
     "hostname": "jenkins-dcp.planx-pla.net",
     "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
-    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/3.2.4/schema.json",
+    "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/gtexdictionary/3.3.8/schema.json",
```

Comment:
latest is 3.3.9

also, the jenkins tests ran against jenkins-genomel not dcp so how are you sure this will fix the jenkins-dcp issues?

Reply:
That was based on a Slack thread.
I will ping you with the details.